app/frontend/src/components/designer/settings/FormEventStreamSettings.vue

```diff
@@ -127,7 +127,7 @@ defineExpose({
             >{{ $t('trans.formSettings.serversLabel') }}:
           </span>
           <span class="ml-2">{{
-            appStore.config?.eventStreamService?.servers
+            appStore.config?.eventStreamService?.consumerservers
```

Comment:
This line wasn't covered in the unit tests before. Would now be a good time to write tests for it?